spec/migrations/20180424141617_azure_backslash_to_forward_slash_spec.rb

```diff
@@ -0,0 +1,116 @@
+require_migration
+
+describe AzureBackslashToForwardSlash do
+  let(:vm_stub) { migration_stub :Vm }
+  let(:stack_stub) { migration_stub :OrchestrationStack }
+  let(:stack_output_stub) { migration_stub :OrchestrationStackOutput }
+  let(:stack_param_stub) { migration_stub :OrchestrationStackParameter }
+  let(:stack_resource_stub) { migration_stub :OrchestrationStackResource }
+  let(:event_stream_stub) { migration_stub :EventStream }
+
+  let!(:vm) do
```

Comment:
test-wise this is really inefficient because *all* of the `let!` are created for every test.  Since each one of these is only used in one test, you could also just not use `let`s at all, and just put the record creation before the `migrate` in each test.  This also makes it easier to read the test because the object is created/migrated/checked all in the same place in the test block.